### PR TITLE
Feat/single rights/413 receipt page implement navigation

### PR DESCRIPTION
**Pull request by Albertlarsen**
## Description
Adds navigation and some texts to SingleRights ReceiptPage

## Related Issue(s)
- #413 

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: src/localizations/no_nn.json [hunk 1]
```diff
@@ -115,6 +115,9 @@
     "missing_delegation_access_title": "Mangler delegeringsrettighet",
     "unknown_title": "Unknown error",
     "new_error_title": "New error",
-    "missing_delegable_right": "Manglar rettigheit"
+    "missing_delegable_right": "Manglar rettigheit",
+    "to_delegation_page": "Gå til delegeringsside",
```
Comment (allinox) [inline src/localizations/no_nn.json:119]:
```suggestion
    "to_delegation_page": "Gå til delegeringssida",
```

File: src/localizations/no_nn.json [hunk 2]
```diff
@@ -115,6 +115,9 @@
     "missing_delegation_access_title": "Mangler delegeringsrettighet",
     "unknown_title": "Unknown error",
     "new_error_title": "New error",
-    "missing_delegable_right": "Manglar rettigheit"
+    "missing_delegable_right": "Manglar rettigheit",
```
Comment (allinox) [inline src/localizations/no_nn.json:118]:
```suggestion
    "missing_delegable_right": "Manglar rett",
```

File: src/features/singleRight/delegate/ReceiptPage/ReceiptPage.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+import classes from './ReceiptPage.module.css';
+
+export const ReceiptPage = () => {
+  const { t } = useTranslation();
+  const navigate = useNavigate();
+
+  return (
+    <PageContainer>
+      <Page>
+        <PageContent>
```
[the remark this answers is not in the document]
Comment (Albertlarsen) [reply src/features/singleRight/delegate/ReceiptPage/ReceiptPage.tsx:18]: As this task was only about implementing navigation I only implemented that 😅 But I'll add it

**Review by allinox — APPROVED** [review]
NaviGREAT! 😁

**Pull request merged**